package.json

```
@@ -12,10 +12,10 @@
     "build:prod": "npm run build -- -c production --base-href=./",
     "build:staging": "npm run codegen:gql && ng build --c staging --base-href=https://catcher-org.github.io/CATcher-staging/",
     "build:prod:web": "npm run codegen:gql && ng build --prod --base-href=https://CATcher-org.github.io/CATcher/",
-    "ng:serve": "ng serve",
-    "ng:serve:web": "npm run postinstall:web && ng serve -o",
+    "ng:serve": "npm run codegen:gql && ng serve",
+    "ng:serve:web": "npm run postinstall:web && npm run codegen:gql && ng serve -o",
```

yea agreed with @dingyuchen so it would be something like 

```suggestion
    "ng:serve:web": "npm run postinstall:web && npm run ng:serve -o",
```